data/applications-menu.appdata.xml.in

```diff
@@ -6,7 +6,7 @@
   <name>Applications Menu</name>
   <summary>Open and search for apps</summary>
   <releases>
-    <release version="2.6.1" date="2020-04-28" urgency="medium">
+    <release version="2.6.1" date="2020-04-29" urgency="medium">
```

Comment:
I think this makes sense since Handy is a pretty big change, and touch screen support is a new feature.

```suggestion
    <release version="2.7.0" date="2020-04-29" urgency="medium">
```